src/commands/createHttpFunction.ts

```diff
@@ -22,60 +21,18 @@ export async function createHttpFunction(context: IActionContext): Promise<void>
 
     const funcApi: AzureFunctionsExtensionApi = await getFunctionsApi(context);
 
-    const endpointName: string = 'endpoint';
     const apiLocation: string = getWorkspaceSetting(apiSubpathSetting, vscode.workspace.workspaceFolders[0].uri.fsPath) || defaultApiLocation;
     const folderPath: string = path.join(vscode.workspace.workspaceFolders[0].uri.fsPath, apiLocation);
 
-    const maxTries: number = 100;
-    let count: number = 1;
-    let newName: string = endpointName;
-
-    while (count < maxTries) {
-        newName = generateSuffixedName(endpointName, count);
-        if (await isNameAvailable(folderPath, newName)) {
-            break;
-        }
-        count += 1;
-    }
-
-    newName = await ext.ui.showInputBox({
-        value: newName, prompt: localize('enterHttpFuncName', 'Provide a unique HTTP Function name for your API'),
-        validateInput: async value => await validateFunctionName(folderPath, value)
-    });
-
     await funcApi.createFunction({
```

Comment:
We might want to check the api version before passing C# as an option, since it won't work unless they have the latest functions extension